src/main/java/org/heigit/ohsome/ohsomeapi/config/SwaggerConfig.java

```diff
@@ -73,6 +73,18 @@ public Docket rawDataDocket() {
                 "Direct access to the full-history of the OSM data"))
         .forCodeGeneration(true).globalResponseMessage(RequestMethod.GET, responseMessages);
   }
+  
+  /** Creates the Swagger2 documentation for the contributions resources. */
+  @Bean
+  public Docket contributionsDocket() {
+    ArrayList<ResponseMessage> responseMessages = defineResponseMessages();
+    return new Docket(DocumentationType.SWAGGER_2).groupName("Contributions").select()
+        .apis(RequestHandlerSelectors.basePackage("org.heigit.ohsome.ohsomeapi.controller.contributions"))
+        .paths(PathSelectors.any()).build().apiInfo(apiInfo()).useDefaultResponseMessages(false)
+        .globalOperationParameters(defineGlobalOperationParams(true))
+        .tags(new Tag("Contributions", "Get the contributed OSM features"))
```

Comment:
Not sure if this would be 100% correct, as we also return a contribution feature triggered by a minor edit (e.g. changing the geometry of a member), right?
If we keep a similar phrasing like in the data-extraction descriptions, we could also go for something like "Direct access to all contributions provided to the OSM data".

Reply:
your suggestion sounds better :+1: